### DISCUSSION TITLE
Repo access

### DISCUSSION THREAD

**Discussion by AprilSweettooth** [Part-II-Computational-Physics/.github#108] [Q&A]
I am not sure why I do not have access to my repo...

**Comment by AustenLamacraft** [comment]
Do you mean admin access (just given you that) or something else?

**Reply by AprilSweettooth** [reply]
Yes, I mean the admin access. It is working now, thanks.

**Reply by AustenLamacraft** [reply]
I just got a message about the large file storage allowance for the organisation, and it seems to come from your project repo. On the other hand, I don't see anything there. Did you try to commit something huge?

**Reply by AprilSweettooth** [reply]
Yes, I am trying to upload my npz file for simulation data. But it seems github only allows 100mb files and mine is around 120-150mb. I tried with LFS but it does not seem working. Is there a way around this? Or can I upload like a zip file and add a line of code in the notebook to unzip it? Thanks!

**Reply by AustenLamacraft** [reply]
It said it was nearly 3GB in the email! I think there is a 1GB total limit in LFS for the whole organisation... Maybe I need to increase the allowance in any case

**Reply by AprilSweettooth** [reply]
I think the total size is indeed around 3GB. It would be very helpful to increase the allowance , thanks.

**Reply by AustenLamacraft** [reply]
OK that's increased but still 3GB is kind of huge. What is it?

**Reply by AprilSweettooth** [reply]
It is a folder of npz files for the generation data for each lenia creatures that I have experimented with. Basically it stores the iteration loop in arrays. I am just trying to record it so that I can run the simulation faster without calculating it every time and I can also extract their entropy in each snapshot.
